lib/queries/getUserFiltersByProvider.ts

```diff
@@ -0,0 +1,29 @@
+import { FetchResult } from "apollo-link";
```

Comment:
I'm a bit confused about the naming in there. The naming implies that what you get is `UserFilters`. What do you think about `getUserIDFromIdentityValue`?

Reply:
Yes, that's a better name. Let's go.